statsmodels/tsa/tests/test_arima_process.py

```diff
@@ -7,7 +7,8 @@
 
 import numpy as np
 from numpy.testing import (assert_array_almost_equal, assert_almost_equal,
-                           assert_equal, assert_raises, assert_, dec)
+                        assert_allclose,
+                        assert_equal, assert_raises, assert_, dec)
```

Comment:
Will do.  flake8 occasionally complains about indentation not being a multiple of 4, seems like a no-win situation.  Not gonna sweat it.